config/routes.rb

```diff
@@ -65,7 +65,7 @@
   get '/phone_confirmation/send' => 'users/phone_confirmation#send_code'
   put '/phone_confirmation' => 'users/phone_confirmation#confirm'
   get '/profile' => 'profile#index'
-  get '/splash' => 'home#index'
+  get '/splash' => 'home#index', as: :splash
```

Comment:
If you want to refer to this thing as splash rather than home should we just call the controller splash?